### DISCUSSION TITLE
Passing arbitrary data to href attributes

### DISCUSSION THREAD

**Discussion by btpy** [voidlabs/mosaico#655] [Q&A]
Hi!
when i'm defining and href attribute with square brackets, e.g.

`<a style="color: #ffffff; -ko-color: @[Color.readability(linkStyle.color, backgroundColor) gt 2 ? linkStyle.color : (Color.isReadable('#ffffff', backgroundColor) ? '#ffffff' : '#000000')]; text-decoration: underline; -ko-color: @linkStyle.color; -ko-text-decoration: @linkStyle.decoration" href="[unsubscribe_link]" data-ko-editable="disiscrivitiText">Unsubscribe</a>`

the href attribute stays the same, but if I change brackets to curly href is mutated to:

`https://domain/path/{unsubscribe_link}`

Is it possible that `href="{unsubscribe_link}"` stays the same?
Thank you

**Comment by bago** [comment]
Have a look there:
https://github.com/voidlabs/mosaico/blob/master/src/js/template-loader.js#L127

Just change that match so to return null when the string is "{something}": otherwise Mosaico thinks this is a relative url and updates it according to the template source url.

**Reply by btpy** [reply]
thanks, that helped!
